src/Desktop/Utility/ExtensionMethods.cs

```diff
@@ -114,7 +114,7 @@ public static Bitmap CaptureBitmap(this A11yElement e)
         public static A11yElement GetParentWindow(this A11yElement el)
         {
             A11yElement prev = null;
-            A11yElement curr = el;
+            A11yElement curr = el ?? throw new ArgumentNullException(nameof(el));
```

Comment:
The analyzer pointed out that `el` isn't validated for null. If `el` is null, the following will happen:
1. `curr` gets set to `null` in line 117
2. The `while` loop in line 118 gets skipped
3. A `NullReferenceException` gets thrown in line 124 when we try to read `curr.ControlTypeId`

This throws an `ArgumentNullException` for clarity